UnitsNet.Tests/BaseDimensionsTests.cs

```diff
@@ -733,13 +733,13 @@ public void CheckBaseDimensionMultiplicationWithForceEqualsMassTimesAcceleration
         [Fact]
         public void CheckToStringUsingMolarEntropy()
         {
-            Assert.Equal("[Length]^2[Mass][Time]^-2[Temperature]^-1[Amount]^-1", MolarEntropy.BaseDimensions.ToString());
+            Assert.Equal("[Length^2][Mass][Time^-2][Temperature^-1][Amount^-1]", MolarEntropy.BaseDimensions.ToString());
```

Comment:
I don't recall exactly where I saw this, but during debugging #1485  I saw values like `L^2MT^-2`, which was a nice and short presentation also I think.

I have no preference here.

Reply:
Yes, I also vaguely remember encountering something like `L^2Θ` (and thinking that I need to fix the `ToString`, but then forgot and so #1485 now has the _simpler_ format). 🤷